spec/acceptance/windows_spec.rb

```diff
@@ -77,7 +75,7 @@ class { 'sensu::agent':
           'backend-url'     => ['wss://sensu-backend:8081'],
           'password'        => 'P@ssw0rd!',
           'name'            => 'sensu-agent',
-          'subscriptions'   => ['base','windows'],
+          'namespace'       => 'default',
```

Comment:
Could you please add a comment block at the top of this that explains our testing methodology and how backends are not supported on windows and because of that we can't test certain things.

Reply:
Comment added at top of file.